### DISCUSSION TITLE
virtio多队列CPS性能测试问题

### DISCUSSION THREAD

**Discussion by BOY332** [baidu/dperf#460] [Q&A] [answered]
我想让多个队列都跑起来，测试虚拟机整体的新建能力，如下我绑定两个cpu，1和2，然后提示我需要配置rss，但是配置rss后又显示端口不支持该配置，请问有什么办法能解决吗，我这个是virtio口，后端vhost是OVS接管的。
![image](https://github.com/baidu/dperf/assets/14010909/cfcd5b19-e1a5-467b-bcd8-be4c23407a84)

![image](https://github.com/baidu/dperf/assets/14010909/9b94d28b-b7e0-4f05-bf22-c5e84e1db067)

**Comment by BOY332** [comment]
![Uploading image.png…]()

**Reply by pengjianzhang** [reply]
rss auto 试试， server 配一个IP

**Reply by BOY332** [reply]
> rss auto 试试， server 配一个IP

![image](https://github.com/baidu/dperf/assets/14010909/21f32975-d72a-496c-b621-e4e769c54334)
![image](https://github.com/baidu/dperf/assets/14010909/ecbf5a24-31c8-467b-ad78-39b949af994d)
这个会报错，virtio口说是不支持RSS，这个特性没协商上

**Comment by BOY332** [comment]
![image](https://github.com/baidu/dperf/assets/14010909/c00d6a2d-7491-4481-a156-45044ab507e3)
会提示这个设备不支持配置RSS

**Comment by pengjianzhang** [comment, marked as answer]
试试 rss auto mq_rx_none

**Reply by BOY332** [reply]
谢谢，这个解决了我的问题

**Reply by pengjianzhang** [reply]
如果喜欢dperf 请加个star 谢谢

**Reply by BOY332** [reply]
yes , I love it

**Comment by BOY332** [comment]
![image](https://github.com/baidu/dperf/assets/14010909/cc90d9ac-9e12-474d-ac2c-fdb3c11bd8a5)
请问一下，我配置了每秒60w新建的cps，dperf上会过一段时间才能打到60w，skOpen我理解就是CPS速率，这样的行为是对的吗？
![image](https://github.com/baidu/dperf/assets/14010909/05a83034-8063-4345-a39f-9fd17754667b)

**Comment by BOY332** [comment]
如果我设置cc为10，表示想要10条长连接，但是显示skCon的值是20多，不是严格相等的呢？

**Reply by pengjianzhang** [reply]
在开启rss的情况下，可能有一些少量误差，你可以试试设置为10K，100K试试
